Examples/Python/GroomUtils.py

```diff
@@ -341,7 +342,8 @@ def reflectMeshes(outDir, seg_list, reference_side):
             seg_out = rename(seg, outSegDir, 'reflect')
 
             mesh = Mesh(seg)
-            mesh.reflect(X, mesh.center()).write(seg_out)
+            center = [mesh.center()[0], mesh.center()[1], mesh.center()[3]]
```

Comment:
arr = mesh.center(); center = [arr[0], arr[1], arr[2]]

In this way, you call mesh.center() only once.